### PR TITLE
#1686 configure accuracy in HashedTimerWheel

### PR DESCRIPTION
**Pull request by zagorulkinde**
Fix issue #1686

### CODE REVIEW

**Comment by slandelle** [comment]
Could you please elaborate on the benefits of this PR? In particular how the defaults are not suited for you.

**Comment by zagorulkinde** [comment]
Hi. In this PR i'd resolved the issue https://github.com/AsyncHttpClient/async-http-client/issues/1686 to decrease delay between RequestTimeout and ListenableFuture's completion. I want to get responses as fast as possible. Also i want to mention that default values now the same as it was before this PR.

**Comment by zagorulkinde** [comment]
Hello @slandelle what do you think?

**Comment by slandelle** [comment]
Why not. Could you please make it more simple and remove the TimeUnit and pass the duration as a number of millis?

**Comment by zagorulkinde** [comment]
removed TimeUnit and passed duration as number of millis;

**Comment by zagorulkinde** [comment]
> Why not. Could you please make it more simple and remove the TimeUnit and pass the duration as a number of millis?

Hey. Any feedback?

File: client/src/main/resources/org/asynchttpclient/config/ahc-default.properties [hunk 1]
```diff
@@ -50,3 +50,5 @@ org.asynchttpclient.shutdownQuietPeriod=2000
 org.asynchttpclient.shutdownTimeout=15000
 org.asynchttpclient.useNativeTransport=false
 org.asynchttpclient.ioThreadsCount=0
+org.asynchttpclient.hashedWheelTimerTickDuration=100
+org.asynchttpclient.hashedWheelTimerSize=512
```
Comment (slandelle) [inline client/src/main/resources/org/asynchttpclient/config/ahc-default.properties]: nit: please add line feed

Comment (zagorulkinde) [reply client/src/main/resources/org/asynchttpclient/config/ahc-default.properties]: done

**Comment by slandelle** [comment]
Thanks!

**Pull request merged**